src/components/attendee/MediaPlayer/MediaPlayer.module.scss

```diff
@@ -10,7 +10,7 @@
   display: flex;
   justify-content: center;
   align-items: flex-end;
-  z-index: 50;
+  z-index: 10;
```

Comment:
This could at least use a comment of why/which elements need specific z ordering